### PR TITLE
Update README.md: more detailed description and instructions

### PR DESCRIPTION
**Pull request by adrianusler**
## Summary by CodeRabbit

- **New Features**
	- Enhanced 3D visualization for molecules and crystal structures.

- **Documentation**
	- Improved Molara installation guide, highlighting virtual environment setup and detailed installation steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,43 @@
...
 [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/Molara-Lab/Molara/main.svg)](https://results.pre-commit.ci/latest/github/Molara-Lab/Molara/main)
 # Molara
+Molara is an open-source program for the 3-dimensional visualization of molecules and crystal structures. These are some of its main features:
+
+1. Import of .xyz, .coord, and POSCAR files
+2. Export of rendered structures as raster graphics
+3. Tools for creating custom molecular and crystal structures
+4. Animation of trajectories
+
+New features will follow soon!
```
[the remark this answers is not in the document]
Comment (ab5424) [reply README.md:12]: Well that's a bit weird, there are no multiple exclamation marks!!1!

**Review by ab5424 — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adrianusler) [reply README.md:12]: good bot.

File: README.md [hunk 2]
```diff
@@ -2,16 +2,43 @@
...
 git clone <this repo>
 cd Molara
 ```
-You first need to create a virtual environment and activate it in order to install molara as follows:
+
+It is advisable to install Molara in a virtual Python environment.
+
+<b>Virtual environment on Linux / Mac:</b>
 ```
+python -m pip install virtualenv
```
Comment (ab5424) [inline README.md:25]: Does one actually need `virtualenv` to create a virtual environment? I thought `venv` was part of standard python since 3.3.

Comment (adrianusler) [reply README.md:25]: I wasn't aware of that, thanks for pointing it out!

**Pull request merged**